mandatory/1-syntax-errors.js

```diff
@@ -1,16 +1,17 @@
 // There are syntax errors in this code - can you fix it to pass the tests?
 
-function addNumbers(a b c) {
+function addNumbers(a, b, c) {
   return a + b + c;
 }
 
-function introduceMe(name, age)
-  return `Hello, my {name}` is "and I am $age years old`;
+function introduceMe(name, age) {
+  return `Hello, my name is ${name} and I am ${age} years old`;
+}
 
 function getTotal(a, b) {
-  total = a ++ b;
+  total = a + b;
```

Comment:
Suggestion: use the let keyword to declare the variable

e.g. `let total = a + b`

The code will work without it but using let scopes the variable to the getTotal function. Without let is there was another function that used a variable called total you could see conflicts.

Reply:
Thanks for the feedback, Michael. So if I don't want total to be used outside of the getTotal function, I need to declare total using let.